_docs/tasks/integrating-services-into-istio.md

```diff
@@ -1,36 +1,207 @@
 ---
 title: Integrating Services into the Mesh
 overview: This task shows you how to integrate your applications with the Istio service mesh.
-            
+
 order: 20
 
 layout: docs
 type: markdown
 ---
 
-This task shows how to do X in a Kubernetes cluster. You'll learn
-how to ...
-
+This task shows how to integrate applications with Istio into a
+Kubernetes cluster. You'll learn how to manually inject the Envoy side
+into deployments using [istioctl
+kube-inject]({{site.bareurl}}/docs/reference/istioctl/istioctl_kube-inject.html)
 
 ## Before you begin
-* Do this.
-* Do this too.
 
-## Doing ...
+This task assumes you have deployed Istio on Kubernetes.  If you have
+not done so, please first complete the [Installation
+Steps]({{site.bareurl}}/docs/tasks/istio-installation.html).
+
+## Injecting Envoy sidecar into a deployment
 
-1. Do this.
-1. Do this next. Possibly read this [related explanation](...).
+Example deployment and service to demonstrate this task. Save this as
+echo.yaml.
```

Comment:
Okay, now we need to decide on what we have to do with the apps. All tasks use bookinfo. This one uses something simple and server side. It doesn't convince me that istio didn't break anything. 

May be do a container list per pod?
May be have our debug app from manager be used as client and our app from manager test be used as a server?